test_mavros/include/sitl_test/test_type.h

```diff
@@ -0,0 +1,35 @@
+/**
+ * @brief SITL test type
+ * @file test_type.h
+ * @author Nuno Marques <n.marques21@hotmail.com>
+ *
+ * @addtogroup sitl_test
```

Comment:
Well I think I will stick to `sitl_test` as it makes more sense. I will do the same on the rest.